cart/cart.js

```diff
@@ -13,10 +13,21 @@ for (let i = 0; i < cart.length; i++) {
     table.appendChild(tr);
 }
 
-const total = calcOrderTotal(cart, foodItems);
+const orderButton = document.querySelector('#order-button');
+orderButton.addEventListener('click', () => {
+    const stringyCart = JSON.stringify(cart, true, 2);
+    // if (cart !== []) {
+    alert(stringyCart);
+    // } else {
+        // alert('You don\'t have any items in your cart. Go back to the Home Page to select an item!');
+    // localStorage.removeItem(CART);
+    localStorage.clear();
+    window.location.href = '/';
```

Comment:
your deployed site isn't routing to home page correctly, but can see it works locally.  Try '../index.html' to route back to your homepage.  Seems like Github wants it to look like a file structure for these links. Let me know if that works!